### PR TITLE
Data cleaning resources dev

### PR DESCRIPTION
**Pull request by wkirui**
# Pull Request Summary 🚀

## What does this PR do? 📝

This change adds the data cleaning guides to the repository and includes a some modification on the page layout as shown here: 
<img width="956" alt="image" src="https://github.com/user-attachments/assets/ab836073-741c-46a0-b841-2f1a287a0824">


## Why is this change needed? 🤔

We need to migrate all existing data cleaning guides into this repository.

## How was this implemented? 🛠️

I copied the data cleaning guides from this [repository](https://github.com/PovertyAction/guides) and modified the code accordingly.

## How to test or reproduce ? 🧪

Running `quarto preview` in the repository folder

## Screenshots (if applicable) 📷

<img width="935" alt="image" src="https://github.com/user-attachments/assets/2cf5f774-a032-4128-84eb-f15f58c4ab05">

## Checklist ✅

- [ ] I have run and tested my changes locally
- [ ] I have limit this PR to less than 1000 lines of code change (if not, explain why)
- [ ] I have updated/added tests to cover my changes (if applicable)
- [ ] I have updated/added requirements to cover my changes (if applicable)
- [ ] I have run linting and formatting on any code changes (if applicable)
- [ ] I have updated the documentation (README, etc.) accordingly
- [ ] I have reviewed and resolved any merge conflict

## Reviewer Emoji Legend

|     |         `:code:`          | Meaning                                                                                                                                                                                                                                         |
| :-: | :-----------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 😃👍💯 | `:smiley:` `:+1:` `:100:` | I like this... <br /><br /> ...and I want the author to know it! This is a way to highlight positive parts of a code review.                                                                                                                    |
| ⭐⭐⭐ |  `:star: :star: :star:`   | Important to fix before PR can be approved... <br /><br /> And I am providing reasons why it needs to be addressed as well as suggested improvements.                                                                                           |
| ⭐⭐  |      `:star: :star:`      | Important to fix but non-blocking for PR approval... <br /><br /> And I am providing suggestions where it could be improved either in this PR or later.                                                                                         |
|  ⭐  |         `:star:`          | Give this some thought but non-blocking for PR approval... <br /><br /> ...and consider this a suggestion, not a requirement.                                                                                                                   |
|  ❓  |       `:question:`        | I have a question. <br /><br /> This should be a fully formed question with sufficient information and context that requires a response.                                                                                                        |
|  📝  |         `:memo:`          | This is an explanatory note, fun fact, or relevant commentary that does not require any action.                                                                                                                                                 |
|  ⛏  |         `:pick:`          | This is a nitpick. <br /><br /> This does not require any changes and is often better left unsaid. This may include stylistic, formatting, or organization suggestions and should likely be prevented/enforced by linting if they really matter |
| ♻️  |        `:recycle:`        | Suggestion for refactoring. <br /><br /> Should include enough context to be actionable and not be considered a nitpick.                                                                                                                        |

### CODE REVIEW

File: docs/data_collection_methods/whatsapp_surveys/getting_started_with_whatsapp_surveys.qmd [hunk 1]
```diff
@@ -13,9 +13,9 @@ author:
 
 **Summary.** This page introduces **WhatsApp surveys** as an alternative data collection method available for IPA projects. While there are multiple ways to approach WhatsApp surveys, the GRDS team recommends <em>Twilio</em> as the platform to implement data collection via WhatsApp. Accordingly, the page provides a quick introduction to Twilio and how to perform initial tasks such as setting up a Twilio console, procuring a phone number, and completing a verification process.
 
-## Introduction
+<!-- ## Introduction -->
```
Comment (NKeleher) [inline docs/data_collection_methods/whatsapp_surveys/getting_started_with_whatsapp_surveys.qmd:16]: :question: Are these lines intentionally commented out?

**Review by NKeleher — COMMENTED** [review]
@wkirui  - Thank you for all of the heavy lifting to migrate the data cleaning content. It's looking great!

Before I approve, given the large reorganization, I think it would be good to walk through the updates with @cpulidoc so that Cristhian can get up to speed on the updates you've made while he was at the India training and weigh in on the best organization of the content. Can the two of you chat in the coming days or next week?

**Comment by NKeleher** [comment]
I'm getting the following 404 error when loading the "Importing into Stata" page. Seems like the path to the `/docs/assets/images/IPA-primary-color-RGB.png` image might need to be fixed.

```bash
❯ just preview-docs
quarto preview
Terminating existing preview server....DONE

Preparing to preview

Watching files for changes
Browse at http://localhost:3434/
GET: /
GET: /docs/data_management/data_cleaning/raw_data_management/
WARN: connection closed before message completed
GET: /docs/data_management/data_cleaning/raw_data_management/importing_into_stata.html
  /docs/assets/images/IPA-primary-color-RGB.png (404: Not Found)
```

**Comment by cpulidoc** [comment]
This draft outline looks great @wkirui! Agree with @NKeleher that we should try to meet soon so you can walk me through your work. I see you're out of office until next Wednesday though, so I sent you an invite next Thursday 5pm (Nairobi). In the meantime, I'll be going over the site and working on content development. Thanks!

**Comment by wkirui** [comment]
> I'm getting the following 404 error when loading the "Importing into Stata" page. Seems like the path to the `/docs/assets/images/IPA-primary-color-RGB.png` image might need to be fixed.
> 
> ```shell
> ❯ just preview-docs
> quarto preview
> Terminating existing preview server....DONE
> 
> Preparing to preview
> 
> Watching files for changes
> Browse at http://localhost:3434/
> GET: /
> GET: /docs/data_management/data_cleaning/raw_data_management/
> WARN: connection closed before message completed
> GET: /docs/data_management/data_cleaning/raw_data_management/importing_into_stata.html
>   /docs/assets/images/IPA-primary-color-RGB.png (404: Not Found)
> ```

I keep getting the same error as well. The path setting to `IPA-primary-color-RGB.png`  is dependent on  "how deep" the current page is in the `docs` folder. So it works for some pages but doesn't work for others. One way we can solve this is by saving this image as a static file in GitHub.

**Comment by NKeleher** [comment]
@wkirui - is this PR ready to merge? If you and @cpulidoc have connected and feel the content is in good shape. Please re-open the review and I can approve.

**Review by NKeleher — APPROVED** [review]
Approving so that we can merge into the main branch.

**Pull request merged**